src/msxfuncs.h

```diff
@@ -6,16 +6,16 @@
 **                 All Rights Reserved. See license information in LICENSE.TXT.
 **  AUTHORS:       L. Rossman, US EPA - NRMRL
 **  VERSION:       1.1.00
-**  LAST UPDATE:   09/29/08
+**  LAST UPDATE:   11/01/10
 ***********************************************************************/
 
 #ifndef MSXFUNCS_H
 #define MSXFUNCS_H
 
 // Define pointers for each group of chemistry functions
-typedef void (*MSXGETRATES)(double *, double *, double * , double *, double *, int, double);

```

Comment:
Looks like this is where the signatures differ? 

Aha! found the cullprit: https://github.com/OpenWaterAnalytics/epanet-msx/commit/ccef38bb43b89454bcadfd7985d48375971f90ed

This commit brought in some changes that were for debugging purposes. I think we can safely drop the VARDEBUG and extra parameters. Just need to make sure that everything is consistent.